gensim/utils.py

```diff
@@ -1164,11 +1164,12 @@ def check_output(stdout=subprocess.PIPE, *popenargs, **kwargs):
     Added extra KeyboardInterrupt handling
     """
     try:
+        cmd = kwargs.get("args")
+        logger.info("COMMAND: %s", cmd)
```

Comment:
1. You should use `logger.debug` here (because it's needed for debug purposes only)
2. Need to logging all input parameters (`popenargs` and `kwargs`)
3. I think now is not necessary to logging some "command specific" data (based on troubles from our users in mailing list), logging all arguments will be enough